src/styles/blocks/card.scss

```
@@ -0,0 +1,117 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  background-color: $background-color;
+  max-width: 200px;
+  transition: transform 300ms;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $font-color;
+    margin: 40px 16px 0;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $details-color;
+    margin: 4px 0 0 16px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    margin: 16px 16px 0;
+
+    &-reviews {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: $font-color;
+    }
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 16px 0;
+    // align-items: flex-end;
+    line-height: 18px;
+
+    &-label {
+      font-weight: 400;
+      font-size: 12px;
+      color: $details-color;
+
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      color: $font-color;
+    }
```

those styles doesn't work 
using `&` you mean that you take name of parent and add some ending, so here you have `card__price-container-label` and you doesn't have element with such class
it should be: 
```suggestion
&__price {

    &-container {
      ...
    }

    &-label {
     ...
    }

    &-value {
      ...
    }
}
```

---

Corrected.